csi/node_server.go

```diff
@@ -442,6 +443,14 @@ func (ns *NodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 			return nil, status.Errorf(codes.InvalidArgument, "unsupported disk encryption format %v", diskFormat)
 		}
 
+		volumeSize, err := util.ConvertSize(volume.Size)
+		if err != nil {
+			return nil, status.Error(codes.Internal, err.Error())
+		}
+		if volumeSize <= crypto.Luks2MinimalVolumeSize {
```

Comment:
"<=" or "<" ?

Reply:
"<=", because I tested 16MiB volume and it still could not be activated.

Reply:
Thank you for the clarification.